mlflow/server/templates/compare_runs.html

```diff
@@ -1,111 +0,0 @@
-{% extends "layout.html" %}
```

Comment:
These have been deprecated for a while, just removing them from the source repo.